test/includes/downtime.php

```diff
@@ -0,0 +1,307 @@
+<?php
+
+/*require_once '../modules/monitoring/includes/datetime.php';
+require_once '../modules/monitoring/includes/downtime.php';*/
```

Comment:
Should these lines be removed?

Reply:
Yes, they probably should. It's for running tests without having to build Ninja.